apps/emqx_durable_storage/src/emqx_ds_storage_bitfield_lts.erl

```diff
@@ -510,6 +517,7 @@ traverse_interval(ITHandle, KeyMapper, Filter, Cutoff, Key, Val, It0, Acc0, N) -
                     Acc = [{Key, Msg} | Acc0],
                     traverse_interval(ITHandle, KeyMapper, Filter, Cutoff, It, Acc, N - 1);
                 false ->
+                    inc_counter(?DS_LTS_COLLISION_COUNTER),
```

Comment:
Since we're using the same counters to track `delete` operations, we might want to add a `inc_counter(?DS_LTS_COLLISION_COUNTER)` to l. 615.

https://github.com/emqx/emqx/blob/17b3b25b07d1b028b10020232325e00c1e98cf86/apps/emqx_durable_storage/src/emqx_ds_storage_bitfield_lts.erl#L602-L615

Reply:
I was thinking about that, but the main purpose of these counters is to tune the LTS layout parameters. From this perspective, delete and next are the same.